tests/unicode/unicodetest.fz

```diff
@@ -0,0 +1,45 @@
+# This file is part of the Fuzion language implementation.
+#
+# The Fuzion language implementation is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as published
+# by the Free Software Foundation, version 3 of the License.
+#
+# The Fuzion language implementation is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+# License for more details.
+#
+# You should have received a copy of the GNU General Public License along with The
+# Fuzion language implementation.  If not, see <https://www.gnu.org/licenses/>.
+
+
+# -----------------------------------------------------------------------
+#
+#  Tokiwa Software GmbH, Germany
+#
+#  Source code of Fuzion test unicodetest
+#
+#  Author: Michael Lill (michael.lill@tokiwa.software)
+#
+# -----------------------------------------------------------------------
+
+# A small test for unicode data related things
+# like string to upper/lower case conversion.
+#
+unicodetest is
+
+  exitCode := 0
+
+  chck(b bool, msg string) is
+    s := if b
+        "PASSED: "
+      else
+        set exitCode := 1
```

Comment:
I don't think so. This runs in the interpreter but for the C-Backend it is stuck in the DFA.

Reply:
![image](https://user-images.githubusercontent.com/88769212/201302297-2c2ac2d8-f04e-4fb1-a36c-6e542bcd4486.png)

Reply:
the problem seems to arise because of `unicode/data.fz` which is ~4000 lines long.
```
lower_case_mappings => mapOf [
    (u32 5031, codepoint 43895),
    (u32 552, codepoint 553),
 ...
```
this is desugared to something like:
```
tmp_arr[0] :=  (u32 5031, codepoint 43895)
tmp_arr[1] :=  (u32 552, codepoint 553),
...
```
And this is where DFA seems to spend ages.